modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZonesSchedulersTest.java

```diff
@@ -258,21 +270,24 @@ void testCancelScaleDownTasksOnStopTimers() {
      * if it is not started and has a delay greater than zero.
      */
     private static void testCancelTask(
-            BiConsumer<Long, Runnable> fn,
+            IgniteTriConsumer<Long, Runnable, Integer> fn,
             Runnable stopTask,
             Supplier<Boolean> isTaskCancelled
     ) {
         CountDownLatch latch = new CountDownLatch(1);
 
-        fn.accept(0L, () -> {
-            try {
-                assertTrue(latch.await(3, TimeUnit.SECONDS));
-            } catch (InterruptedException e) {
-                throw new RuntimeException(e);
-            }
-        });
+        fn.accept(0L,
+                () -> {
+                    try {
+                        assertTrue(latch.await(3, TimeUnit.SECONDS));
+                    } catch (InterruptedException e) {
+                        throw new RuntimeException(e);
+                    }
+                },
+                STRIPE_0);
 
-        fn.accept(1L, () -> {});
+        fn.accept(1L, () -> {
+        }, STRIPE_1);
```

Comment:
this also needs correct formatting, and the same cases below

Reply:
Fixed.